po/fr.po

```diff
@@ -78,7 +78,7 @@ msgstr "Cet article est placé sous la licence {{ license }}"
 
 msgid "One like"
 msgid_plural "{{ count }} likes"
-msgstr[0] "{{ count }} personne aime cet article"
+msgstr[0] "Une personne aime cet article"
```

Comment:
This fail when there are 0 likes, as in french 0 is singular it is treated by this rule. If a post is liked by nobody it will still show "Une personne aime cet article"

Reply:
Thanks for your feedback. My bad, I thought `msgstr[0]` was for the singular `msgid` and `msgstr[1]` for `msgid_plural`. Is there a way to have a string for zero, a string for one, and a string for *n*?

French behaves a bit strangely; whereas the number 1 is singular, the number 0 is plural unless we replace it with an expression like “_Aucun⋅e_” (None) or “_Personne_” (No one) to make it singular.

> _0 personne**s** aime**nt** cet article_ or _Personne n'aime cet article_
> _Une personne aime cet article_
> _2(+) personne**s** aime**nt** cet article_

I would avoid “_Personne n'aime cet article_” since it translates to “No one likes this article” which could make authors feel quite bad, unless we use the past tense “_Personne n'a aimé cet article_” (“No one liked this article”), which suggests a “_pour l'instant_”/“yet”.

Would something like that work?

```
msgid "One like"
msgid_plural "{{ count }} likes"
msgstr[0] "Personne n'a aimé cet article"
msgstr[1] "Une personne aime cet article"
msgstr[2] "{{count}} personnes aiment cet article"
```

It would be somehow inconsistent though to use the past tense for the zero-case and the present tense for the others. Perhaps we could consider using the past tense everywhere?

Reply:
Zero is not plural in french, there are some situation where we use zero with a plural ("zéro sucres ajoutés") but the general case is zero is singular, ex "[zéro faute d'orthographe]( https://fr.wiktionary.org/wiki/z%C3%A9ro#Adjectif_num%C3%A9ral)"

To choose what is plural and singular, the i18n engine look at the header `Plural-Forms` (line 13), for some plural it's simply `n != 1` (English, German...), some are more complex (take a look at pl.po if you want, in that language there are 2 plurals and one singular)

Even if I'm a native french speaker I've a hard time knowing what might be hurting peoples, I'm not exactly of the social kind. I guess you can do whatever seems best to translate without hurting people's feelings. (If you want to translate with more than 2 forms, i.e. zero, one and more, tell me and I will send you the corresponding `Plural-Forms`, but then you will have to update every msgstr[X] to the new plural, you did not update reshages translation previously)